docs/core/whats-new/dotnet-9/libraries.md

```diff
@@ -20,6 +20,10 @@ ReadOnlySpan<byte> bytes = ...;
 string encoded = Base64Url.EncodeToString(bytes);
 ```
 
+## BinaryFormatter
+
+.NET 9 completes the removal of <xref:System.Runtime.Serialization.Formatters.Binary.BinaryFormatter> from the .NET runtime. The APIs are still present, but their implementation always throws an exception, regardless of project type. For more information about your options if this removal affects you, see [BinaryFormatter migration guide](../../../standard/serialization/binaryformatter-migration-guide/index.md).
```

Comment:
> completes the removal

This makes it sound like it was some long publicized initiative. I don't think most readers will realize / connect with that.

```suggestion
.NET 9 removes  <xref:System.Runtime.Serialization.Formatters.Binary.BinaryFormatter> from the .NET runtime. The APIs are still present, but their implementation always throws an exception, regardless of project type. For more information about your options if this removal affects you, see [BinaryFormatter migration guide](../../../standard/serialization/binaryformatter-migration-guide/index.md).
```